subiquity/ui/views/filesystem/partition.py

```diff
@@ -567,7 +572,8 @@ def __init__(self, parent, device):
             initial.update(initial_data_for_fs(fs))
         elif not isinstance(device, Disk):
             initial['fstype'] = 'ext4'
-        self.form = PartitionForm(self.model, 0, initial, None, device)
+        self.form = PartitionForm(
+            self.model, 0, initial, None, device, 1 << 20)
```

Comment:
A constant would be nice, to help ensure it doesn't skew from the value at line 384.